### PR TITLE
Env config

### LINKED ISSUE #26: Read redis config and server listen address/port from env variables

### PR DESCRIPTION
**Pull request by draperunner**
Fixes #26

### CODE REVIEW

File: coffee/config.py [hunk 1]
```diff
@@ -1,16 +1,11 @@
...
-    'DEBUG': config.getboolean('coffee_server', 'debug'),
-    'REDIS_DB': int(config.get('coffee_server', 'redis_db')),
-    'REDIS_HOST': config.get('coffee_server', 'redis_host'),
-    'REDIS_PORT': int(config.get('coffee_server', 'redis_port')),
+    'DEBUG': bool(os.getenv('DEBUG', True)),
+    'REDIS_DB': int(os.getenv('REDIS_DB', 1)),
+    'REDIS_HOST': os.getenv('REDIS_HOST', '127.0.0.1'),
+    'REDIS_PORT': int(os.getenv('REDIS_PORT', 6379)),
+    'REDIS_PW': os.getenv('REDIS_PW', None),
+    'SERVER_HOST': os.getenv('SERVER_HOST', '0.0.0.0'),
```
Comment (eirsyl) [inline coffee/config.py]: I think you should bind to `127.0.0.1` by default 👍

Comment (draperunner) [inline coffee/config.py]: I set it to 0.0.0.0 because that is apparently the Flask default?

**Comment by eirsyl** [comment]
Nice! Looks good to me 😄

**Comment by ekmartin** [comment]
Same - merge away :)

**Pull request merged**